app/javascript/admin/controllers/config_controller.js

```diff
@@ -291,13 +291,12 @@ export default class ConfigController extends Controller {
     document
       .querySelectorAll('[data-enable-auth="true"]')
       .forEach((provider) => {
-        const providerName = provider.dataset;
-        const providerOfficialName = provider.dataset;
+        const providerName = provider.dataset.providerName;
```

Comment:
refactor (non-blocking): prefer destructuring.

```suggestion
        const { providerName } = provider.dataset;
```